package.json

```diff
@@ -22,11 +22,12 @@
     "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
     "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
-    "pre-commit": "lint-staged"
+    "precommit": "lint-staged"
```

Comment:
The reaction-force task ran the tests iirc, which was redundant because you’re probably already running tests during dev and thus would slow down making commits considerably for no good reason.